docs/examples/scf_to_cps/imputing-from-scf-to-cps.md

```diff
@@ -394,265 +419,525 @@ To assess the imputation results, a comparison the distribution of wealth in the
+        mode="lines",
+        line=dict(color=scf_mean_color, width=2, dash="dot"),
+        name=f"SCF mean: ${10**scf_log_mean:,.0f}",
+    ))
+    fig.add_trace(go.Scatter(
+        x=[cps_log_mean, cps_log_mean],
+        y=[0, 20],
+        mode="lines",
+        line=dict(color=cps_mean_color, width=2, dash="dot"),
+        name=f"CPS mean: ${10**cps_log_mean:,.0f}",
+    ))
+
+    # layout
     fig.update_layout(
-        title_text="Comparison of Original and Imputed Networth Distributions",
+        title=title if title else "Log-transformed net worth distribution comparison",
+        xaxis_title="Net worth",
+        yaxis_title="Frequency",
         height=PLOT_CONFIG["height"],
-        width =PLOT_CONFIG["width"],
-        legend_tracegroupgap=180
+        width=PLOT_CONFIG["width"],
+        barmode="overlay",
+        bargap=0.1,
+        legend=dict(
+            x=0.01, y=0.99,
+            bgcolor="rgba(255,255,255,0.8)",
+            bordercolor="rgba(0,0,0,0.3)",
+            borderwidth=1,
+            orientation="v",
+            xanchor="left",
+            yanchor="top",
+        ),
     )
 
-    # 8. Zoom x-axis to drop extreme outliers
-    fig.update_xaxes(range=[0, upper_limit], row=1, col=1, title_text="Networth")
-    fig.update_xaxes(range=[0, upper_limit], row=2, col=1, title_text="Networth")
-    fig.update_yaxes(title_text="Frequency", row=1, col=1)
-    fig.update_yaxes(title_text="Frequency", row=2, col=1)
+    # custom ticks
+    tick_values = [-6, -4, -2, 0, 2, 4, 6, 8]
+    tick_labels = [
+        "$" + format(10**x if x >= 0 else -(10**abs(x)), ",.0f")
+        for x in tick_values
+    ]
+    fig.update_xaxes(tickvals=tick_values, ticktext=tick_labels)
 
     return fig
+
+weights_col = cps_data["household_weight"].values
+weights_normalized = weights_col / weights_col.sum()
+imputed_data_weighted = imputed_data.sample(
+    n=len(imputed_data),
+    replace=True,
+    weights=weights_normalized,
+).reset_index(drop=True)
+
+
+plot_log_transformed_distributions(scf_data_weighted, imputed_data_weighted).show()
 ```
 
-![png](./scf_to_cps_networth_distribution.png)
+![png](./autoimpute_best_model_imputations.png)
+
+The logarithmic transformation provides a clearer view of the wealth distribution across its entire range. By logarithmically scaling the data, the extreme values are compressed while expanding the visibility of differences in the lower and middle portions of the distribution.
+
+This transformation is particularly valuable for wealth data, where values can span many orders of magnitude. The plot above, shows how closely the imputed CPS wealth distribution matches the original SCF distribution in terms of shape and central tendency after the imputation performed by the QRF model. The vertical lines marking the mean and median values help gauge how these statistical properties have been preserved through the imputation process.
+
+## Comparing with other methods
 
 ```python
```

Comment:
Could we make this a notebook, with actual code cells?

Reply:
Ah yes, this would take a long time to run so let's keep as a markdown file (but keep the actual notebook in the paper folder)